__tests__/commands/install/integration.js

```diff
@@ -737,6 +737,10 @@ test.concurrent('install will not overwrite files in symlinked scoped directorie
   });
 });
 
+test.concurrent('install of scoped package with subdependency conflict should pass check', (): Promise<void> => {
+  return runInstall({}, 'install-scoped-package-with-subdependency-conflict');
```

Comment:
nit: could you add an explicit `check` call for this test?
Even though all install script have implicit check running it is better from maintainability point of view to have an explicit expectation from the test run, e.g.:
```
    let allCorrect = true;
    try {
      await check(config, reporter, {integrity: false}, []);
    } catch (err) {
      allCorrect = false;
    }
    expect(allCorrect).toBe(true);
```